### PR TITLE
Adding survey_id to the resources

### PR DESCRIPTION
**Pull request by AlxEnashi**

### CODE REVIEW

File: stamp_service/fits2png.py [hunk 1]
```diff
@@ -6,8 +6,12 @@
 
 
 def _read_compressed_fits(compressed_fits_file):
-    fits = io.BytesIO(compressed_fits_file)
-    return fio.open(gzip.open(fits))[0]
+    try:
+        fits = io.BytesIO(compressed_fits_file)
+        return fio.open(gzip.open(fits))[0]
+    except Exception as e:
```
Comment (pcastelln) [inline stamp_service/fits2png.py:12]: Why not go only for the specific expected error? I think it is OSError

Comment (AlxEnashi) [reply stamp_service/fits2png.py:12]: will do. and yes its os error

File: stamp_service/search.py [hunk 2]
```diff
@@ -8,16 +8,20 @@
 
 
 class S3Searcher:
-    def init(self, bucket_name, client=None):
+
+    buckets_dict = {}
```
Comment (pcastelln) [inline stamp_service/search.py:12]: Why have this as a class attribute? Seems like just initializing in __init__ should be enough, unless I'm missing something

Comment (AlxEnashi) [reply stamp_service/search.py:12]: was a place holder, in the init its enough

File: stamp_service/utils.py [hunk 3]
```diff
@@ -45,4 +45,4 @@ def reverse_candid(candid):
     candid : int or str
```
[the remark this answers is not in the document]
Comment (AlxEnashi) [reply stamp_service/utils.py:45]: right

[the remark this answers is not in the document]
Comment (AlxEnashi) [reply stamp_service/utils.py:45]: restored the cast, candid is int or str in some situations, this avoid any problems with that.

**Pull request merged**